scanner/engine/python.in.cpp

```diff
@@ -93,9 +99,15 @@ BOOST_PYTHON_MODULE(libscanner) {
   class_<FailedVideo>("FailedVideo", no_init)
     .def_readonly("path", &FailedVideo::path)
     .def_readonly("message", &FailedVideo::message);
+  class_<proto::Result>("Result", no_init)
```

Comment:
What do you mean? I tried just returning the `proto::Result`, but that didn't work.